src/elona/save_update.cpp

```diff
@@ -638,8 +641,8 @@ void _update_save_data_4(const fs::path& save_dir, int serial_id)
 
         const auto is_invs1 = entry.path().filename() == "inv.s1" ||
             entry.path().filename() == "g_inv.s1";
-        const auto begin = is_invs1 ? 0 : 1320;
-        const auto end = is_invs1 ? 1320 : 5480;
+        const auto begin = is_invs1 ? 0 : ELONA_OTHER_INVENTORIES_INDEX;
```

Comment:
In this file, `save_update.cpp`, magic numbers should be hard-coded. These constants may be changed some day, but these migration routines must be able to handle old saves.